app/src/main/res/values/terms_of_service.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
```

Comment:
Two things:
1. There are also spacing issues here similar to the privacy policy--please fix.
2. The privacy policy link should be properly linkified (and should probably link to the privacy policy page using the custom handler)

Further, do the two policies look correct if you keep the newlines between the elements? It's very hard to review these files when they're one line, and future changes will be nearly impossible to spot since the whole line will be considered changed.

Reply:
Do I need to break these lines? I am not understanding what to do here.

Reply:
Done.